hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -62,6 +63,8 @@ protected Object run() throws Exception {
 
     if (HystrixCommand.class.isAssignableFrom(method.getReturnType())) {
       return hystrixCommand;
+    } else if (Observable.class.isAssignableFrom(method.getReturnType())) {
+      return hystrixCommand.toObservable();
```

Comment:
Great idea looking at RxNetty.  To avoid a 3rd compat concern, I'd probably
defer to cold unless @NiteshKant has cause to believe RxNetty will warm up
in the future.